migrations/versions/20211207_153929_removed_time_from_reservations.py

```diff
@@ -0,0 +1,28 @@
+"""Removed time from reservations.
```

Comment:
Can you delete this whole file and push again so it's just not included since it's just deleting something you had locally?